### ISSUE TITLE
declare/require functionality

### ISSUE THREAD

**Issue by LockeBirdsey** [AdrianJohnston/Phorcys#1]
We need a way to easily create Panes in separate files which would require the use of requirejs, browersify, etc. However, these either require Node.js or prevent passing of arbitrary values. A solution to the latter problem would be creating everything as modules (which would require a silly amount of work). 

Using order HTML script tags could work, however will break when circular dependencies are encountered. 

**tl;dr** Too annoying to use requirejs or others, will continue messing around in a single file :(
